packages/core/tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "outDir": "./lib",
     "rootDir": "./src",
     "strict": true,
+    "sourceMap": true,
```

Comment:
Does source maps works? Helped to resolve your problem?

Reply:
Local debug is working with it.